bitnami/zookeeper/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 name: zookeeper
-version: 0.0.3
+version: 0.0.4
```

Comment:
I don't see any other change in zookeeper, did you forget to commit it?

Reply:
That didn't require changes, reverting it.